README.md

```diff
@@ -1,55 +1,361 @@
-| v1.5.1                              |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: | :warning:          |                    |                    |                    |           |
-| v1.6.0                              |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :warning:          |                    |                    |           |
-| v1.7.0                              |                    |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: | :warning:          |                    |           |
-| v1.8.0                              |                    |                    |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: | :warning:          |           |
-| v1.9.0                              |                    |                    |                    |                    |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: | :warning: |
-
-- :white_check_mark: Compatible
-- :warning: Has issues
-- :x: Incompatible
-
-### Warning
-
-- :warning: : module version: `v1.6.0` and Kubernetes Version: `1.20.x`. It works as expected. Marked as warning
-because it is not officially supported by [SIGHUP](https://sighup.io).
-- :warning: : module version: `v1.7.0` and Kubernetes Version: `1.21.x`. It works as expected. Marked as warning
-because it is not officially supported by [SIGHUP](https://sighup.io).
-- :warning: : module version: `v1.8.0` and Kubernetes Version: `1.22.x`. It works as expected. Marked as warning
-because it is not officially supported by [SIGHUP](https://sighup.io).
-- :warning: : module version: `v1.9.0` and Kubernetes Version: `1.23.x`. It works as expected. Marked as warning
-because it is not officially supported by [SIGHUP](https://sighup.io).
+| Kubernetes Version |   Compatibility    |                        Notes                        |
+| ------------------ | :----------------: | --------------------------------------------------- |
+| `1.20.x`           | :white_check_mark: | No known issues                                     |
+| `1.21.x`           | :white_check_mark: | No known issues                                     |
+| `1.22.x`           | :white_check_mark: | No known issues                                     |
+| `1.23.x`           |     :warning:      | Conformance tests passed. Not officially supported. |
+
+Check the [compatibility matrix][compatibility-matrix] for additional informations about previous releases of the modules.
```

Comment:
informations -> information (is an uncountable)

Reply:
I feel like you told me this like 10 times. thanks